contracts/zksync/test/set_order.sh

```diff
@@ -11,18 +11,29 @@ BRIDGE_AMOUNT_WEI=$(echo "scale=0; $VALUE_WEI - $FEE" | bc)
 BRIDGE_AMOUNT_ETH=$(echo "scale=18; $BRIDGE_AMOUNT_WEI / 10^18" | bc)
 # BRIDGE_AMOUNT_WEI=$(printf "%.0f" "$BRIDGE_AMOUNT_WEI")
```

Comment:
This file must use the same syntax than `.github/workflows/scripts/set_order.sh`